src/models/backingImage.js

```diff
@@ -26,10 +27,12 @@ export default {
   subscriptions: {
     setup({ dispatch, history }) {
       history.listen(location => {
-        dispatch({
-          type: 'query',
-          payload: location.pathname === '/backingImage' ? queryString.parse(location.search) : '',
-        })
+        if (enableQueryData(location.pathname, 'backingImage')) {
+          dispatch({
+            type: 'query',
+            payload: location.pathname.startsWith('/backingImage') ? queryString.parse(location.search) : {},
```

Comment:
`enableQueryData` already does the pathname check. Do we still need this ternary expression? I think we just need `payload: queryString.parse(location.search)`

The same for others.

Reply:
`enableQueryData` is only used to determine which http request the current page needs to load. It doesn't determine if the current request needs to send parameters to filter data. For example, In the volume page we needs to load node data and volume data. Because it’s in volume page so only the volume data needs to be filtered, node data don't need to be filtered. So I need this judgment to get the full node data.

Reply:
You're right 👍